ionic_user_interface/src/components/VoteButton.vue

```diff
@@ -117,13 +128,19 @@ export default defineComponent({
   justify-content: center;
   font-size: 16px;
   border-radius: 10px;
-  border: 2px solid grey;
+  border: 2px solid var(--ion-color-danger);
   padding: 5px 7px;
   align-self: center;
 }
 
+.vote-button-component:hover {
+  background-color: var(--ion-color-danger-shade);
```

Comment:
![image](https://user-images.githubusercontent.com/46164638/126278311-9987f4a8-cad2-45ea-8a89-d2ca738f5a44.png)
Hover bgColor is still blending in with the text/outline